src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int NUMBER_OF_BALLS = 3;
+
+    public static void main(String[] args) {
+
```

Comment:
redundant empty line

Reply:
"main" should be separeted from previous statement!